sbin/build.sh

```diff
@@ -237,8 +237,9 @@ configureVersionStringParameter() {
     BUILD_CONFIG[VENDOR_BUG_URL]="https://gitee.com/openeuler/bishengjdk-11/issues"
     BUILD_CONFIG[VENDOR_VM_BUG_URL]="https://gitee.com/openeuler/bishengjdk-11/issues"
   fi
-
-  addConfigureArg "--with-vendor-name=" "\"${BUILD_CONFIG[VENDOR]}\""
+  if [ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" != 8 ]; then
+    addConfigureArg "--with-vendor-name=" "\"${BUILD_CONFIG[VENDOR]}\""
```

Comment:
Does variant OpenJ9 have a BUILD_CONFIG[VENDOR] set?

Reply:
yeah in our builds it's set as `Eclipse Foundation`

Reply:
ok. Wasn't obvious as I don't see it set above.